src/components/Menu.scss

```diff
@@ -0,0 +1,73 @@
+.Menu{
+  position: absolute;
+  width: 398px;
+  height: 896px;
+  left: 16px;
+  top: 0px;
+  
+  /* White */
+  background: #FFFFFF;
+  box-shadow: -4px 2px 8px rgba(0, 46, 93, 0.08);
+}
+
+.Title {
+  width: 97px;
+  height: 28px;
+
+  font-family: 'Arial';
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 28px;
+  
+  padding-top: 43px;
+  padding-bottom: 26px;
+
+  margin-left: auto;
+  margin-right: auto;
+
+  color: #058CCC;
+}
+
+.MainMenu {
+  // TODO: Fix if list is long, make it scroll
+  @media (min-width: 600px) {
+    .wrapper {
+        display: grid;
+        grid-template-columns: 250px 1fr;
+        gap: 1rem;
+    }
+  }
+  width: 398px;
+  height: 720px;
+  left: 16px;
+  top: 97px;
+  background: #F2FBFF;
+}
+
+#LogOut * {
```

Comment:
I'd recommend using class names instead of id tags to style. Following the [Single Responsibility Principle](https://en.wikipedia.org/wiki/Single-responsibility_principle), I'd recommend using ids and classnames both for styling.

My recommendation would be to use classnames for styling and ids for scripting